docs/en/operations/table_engines/join.md

```diff
@@ -66,12 +66,12 @@ SELECT joinGet('id_val_join', 'val', toUInt32(1))
 
 ### Selecting and Inserting Data
 
-You can use `INSERT` to add data to the table. For the `ANY` strictness, data for duplicated keys are ignored. For the `ALL` strictness, all rows are kept.
+You can use `INSERT` to add data to tables. With the `ANY` strictness, data for duplicate keys are ignored. With the `ALL` strictness, all rows are added.
```

Comment:
I'd once again mention that first sentence applies only to this engine, out of context it sounds misleading